reactive/kubernetes_master.py

```diff
@@ -1410,12 +1417,47 @@ def push_service_data():
         kube_api.configure(kubernetes_master.STANDARD_API_PORT)
 
 
-@when("certificates.available", "kube-api-endpoint.available", "cni.available")
+@when("leadership.is_leader")
+@when("endpoint.lb-provider.available")
+@when_not("kubernetes-master.sent-lb-request")
+def request_load_balancer():
+    """Request a LB from the related provider."""
+    lb_provider = endpoint_from_name("lb-provider")
+    req = lb_provider.get_request("api-server")
+    req.protocol = req.protocols.tcp
+    port = kubernetes_master.STANDARD_API_PORT
+    req.port_mapping = {port: port}
+    req.public = True
+    if not req.health_checks:
+        req.add_health_check(
+            protocol=req.protocols.http,
+            port=8080,
+            path="/livez",
```

Comment:
kube-apiserver is listening on https port 6443, right? What's listening on http port 8080?

Reply:
Ah, good catch.  I'm not sure where I got that from.